### DISCUSSION TITLE
Alternative to the current shoving mechanic

### DISCUSSION THREAD

**Discussion by Ranzarth** [unitystation/unitystation#7521] [Ideas]
## Description
 Instead of how people are shoved to the ground currently, we should instead implement the mechanic TG currently uses. 

Instead how shoving should work is that the person you want to shove has to be against a wall or table, which will guarantee they will fall to the ground every time, and if they are not against a wall or table, then it will be impossible to shove them to the ground. Also if against a table, they will be pushed onto the table as well as being downed. 

This is actually one of the newer mechanics that they added in TG that I actually liked as it requires tactics instead of just spamming disarm to shove someone to the ground.

**Comment by PerfectTangent** [comment]
converting to discussion
